src/screens/Home/Home.tsx

```diff
@@ -1,7 +1,59 @@
+import GamepadRoundedIcon from '@mui/icons-material/GamepadRounded';
+import { Box, Fab, Typography } from '@mui/material';
+import Button from '@mui/material/Button';
+import { NavLink } from 'react-router-dom';
+
+import { ROUTES } from '../../routes/Routes';
+import { style } from './Home.style';
+
 export const Home = () => {
     return (
-        <>
-            <div>Home</div>
-        </>
+        <Box sx={style.pageBox}>
+            <Box sx={style.logoBox}>
+                <Typography component={'h1'} variant='h2' color='white'>
+                    TeamUp
+                </Typography>
+                <Fab color='primary' aria-label='logo' sx={style.fabLogo}>
+                    <GamepadRoundedIcon sx={{ fontSize: '48px' }} />
+                </Fab>
+            </Box>
+            <Typography variant='h6' color='white' sx={style.firstTypography}>
+                Don't wait anymore, find <br /> out your game mate now
+            </Typography>
+            <NavLink to={ROUTES.REGISTER} style={style.linkText}>
```

Comment:
I wonder if this navlink is necessary, we have button under so we can get rid of this tag and just add onclick to button, @Veryyapeee right?

Reply:
1. Adding onClick is creating uneccesaary function and you have to get navigation object to run the method

2. Link component is semantic, just button is not in this case

Reply:
So now it is as you say :) if think so